### PR TITLE
Migration to Webpack + movement of core code to the visual + latest A…

### PR DESCRIPTION
**Pull request by EugeneElkin**
…PI + packages update

### CODE REVIEW

File: src/visual.ts [hunk 1]
```diff
@@ -103,7 +108,21 @@ export default class TimelineStoryteller implements IVisual {
      */
     public update(options: VisualUpdateOptions): void {
         const dv = this.dataView = options.dataViews && options.dataViews[0];
+        
         if (dv && dv.categorical) {
+
+            const visualSelection = d3.select(".timeline_storyteller");
```
Comment (uve) [inline src/visual.ts:114]: replace .timeline_storyteller  by this.element

**Review by uve — CHANGES_REQUESTED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,22 @@
+## 2.0.4
+* Filter applying fix
+
+## 2.0.3
+* Additional items for context menu
+
+## 2.0.2
+* Switched to API 3.4
```
[the remark this answers is not in the document]
Comment (EugeneElkin) [reply CHANGELOG.md]: Done

File: capabilities.json [hunk 3]
```diff
@@ -2,64 +2,140 @@
...
       "name": "facet",
-      "kind": 0,
+      "kind": "Grouping",
       "displayName": "Facet"
-    }, {
+    },
+    {
       "name": "content_text",
-      "kind": 0,
+      "kind": "Grouping",
```
[the remark this answers is not in the document]
Comment (EugeneElkin) [reply capabilities.json:10]: Cannot be done. It breaks compatibility.

File: src/visual.ts (hunk 1 above)
Comment (EugeneElkin) [reply src/visual.ts]: Done

**Review by uve — APPROVED** [review]

**Pull request merged**